src/components/MainPage/Importers/Importer.vue

```diff
@@ -105,42 +125,29 @@ export default {
     },
     onProgress({ percent, message, error }) {
       if (percent === undefined) throw new Error('You must to set \'percent\' value for progress');
-
       this.percentage = Math.floor(percent * 100);
-
       if (message) {
         this.step = message;
         this.$logger.info(message);
       }
-
       if (error) {
-        this.$logger.error(error.message);
+        this.$logger.error(error);
```

Comment:
I don't know...
I did this because if you put the whole `error` object, the logger can print the whole object, too. And if there is a circular reference in the `error` object....